src/pymovements/events/properties.py

```diff
@@ -143,6 +170,18 @@ def location(
 ) -> pl.Expr:
     """Location of an event.
 
+    For method ``mean`` the location is calculated as:
+
+    .. math::
+        \\text{Location} = \\frac{1}{n} \\sum_{i=1}^n \\text{position}_i
+
+    For method ``median`` the location is calculated as:
+
+    .. math::
+        \\text{Location} = \\text{median} \\left(\\text{position}_1, \\ldots,
```

Comment:
here again, maybe instead of 
```math
\text{Location} = \text{median} \left(\text{position}_1, \ldots, \text{position}_n \right)
```
maybe write that?

```math
\text{Location} = \begin{pmatrix} \text{median}(x) \\ \text{median}(y) \end{pmatrix}
```

Reply:
see comment above